src/Core/Ordering/Command/AddProductToCart.php

```
@@ -3,15 +3,28 @@
+    /**
+     * @param class-string<AddItemStrategyContract> $strategy
+     */
     public function __construct(
         public UuidInterface $cartUuid,
         public string $sku,
         /** @var positive-int */
         public int $quantity,
+        public string $strategy = MergeSameSkuItemsStrategy::class,
     ) {
+        if (!is_a($this->strategy, AddItemStrategyContract::class, true)) {
+            throw new \InvalidArgumentException(sprintf(
```

PHPMD (2.15.0)

Missing class import via use statement (line '23', column '23'). (undefined)

(MissingImport)